src/ray/common/ray_config_def.h

```diff
@@ -50,6 +50,11 @@ RAY_CONFIG(int64_t, max_direct_call_object_size, 100 * 1024)
 /// this many milliseconds.
 RAY_CONFIG(int64_t, initial_reconstruction_timeout_milliseconds, 10000)
 
+/// The maximum duration that workers can hold on to another worker's lease
+/// for direct task submission until it must be returned to the raylet.
+/// TODO(edoakes): what should we default to?
```

Comment:
@ericl thoughts?

Reply:
Sounds great for now. Feel free to remove the TODO.

You can note that high values will lead to more unfairness, but potentially higher throughput since the scheduler does less work.